package.json

```diff
@@ -29,6 +29,10 @@
         "test-sirius": "cd internal/sirius && go test",
         "test-server": "cd internal/server && go test",
         "cypress": "cypress open",
+        "cypress-headless": "cypress run -b chrome",
+        "cypress-for-m1-build": "docker-compose -f docker/docker-compose.ci.yml build deputy-hub json-server",
+        "cypress-for-m1-up": "docker-compose -f docker/docker-compose.ci.yml up -d deputy-hub json-server",
+        "cypress-build-down": "docker-compose -f docker/docker-compose.ci.yml down",
```

Comment:
I don't think these are m1 specific, and could the build command be removed if the up used `--build`?